### ISSUE TITLE
Remove JavaScript

### ISSUE THREAD

**Issue by Kyiro** [tsoding/todo.asm#2]
I'm proposing a move from client-side JavaScript to a state-of-the-art [HTML form](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form). This will add more blazingly fast 🚀 assembly code and allow it to run on my Sony Ericsson cell phone or IE8 as they don't have the "modern" APIs like `fetch`.
For reference, this is how the app currently handles sending a message to the server:
```asm
index_page_footer db "</ul>", 10
                  db "<input id='todoText' type='text' onkeypress='handleKeyPress(event)'><input type='submit' value='add' onclick='addTodo()'>", 10
                  db "<script>", 10
                  db "    async function handleKeyPress(event) {", 10
                  db "        if (event.key == 'Enter') addTodo();", 10
                  db "    }", 10
                  db "    async function addTodo() {", 10
                  db "        await fetch('/', { method: 'POST', body: todoText.value });", 10
                  db "        location.reload();", 10
                  db "    }", 10
                  db "    todoText.focus();", 10
                  db "</script>", 10```

**Comment by rexim** [comment]
Feel free to submit a PR for that

**Comment by Kyiro** [comment]
> Feel free to submit a PR for that

On it.

**Comment by somrandom** [comment]
html form also has a get method 

                    index_page_footer db "</ul>", 10
                                      db "<form method='get'>", 10
                                      db "  <input type='text' name='todo'>", 10
                                      db "  <input type='submit', value='add'>", 10
                                      db "</form>", 10

I had partial succsess with this but am not comfortable enough with fasm, i managed to get this in my todo though:
/?todo=asdf HTTP/1.1 Host: localhost:6969 Connection: keep-alive Upgrade-Insecure-Requests: 1 User-Agent.......

**Comment by rexim** [comment]
Fixed in 47dac309d524ede09de4bc54eab27da39dec4bfb
